problems/objects.js

```diff
@@ -151,15 +192,31 @@ const authorScores = () => {};
  * @returns {string} The full name of person with best score.
  */
 
-const bestScore = () => {};
+const bestScore = (submissions) => {
+  
+  let highestScore = -Infinity;
+  let score = submissions[i]["score"]
```

Comment:
i is not defined on this line

Reply:
What's the goal of the score? Is there something else we can track or save instead?